### DISCUSSION TITLE
[Unstable] Catastrophic debugdraw framerate

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14253] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Just open debugdraw in an outpost and zoom out a bit

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/fa937409-6cd1-4caa-a9a9-8523785e6c1d)

### Reproduction steps

1

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Issue ticket here #14335.

**Comment by Jasontti** [comment]
Tested a fix for this, no issues found.
Performance is on par with current stable release.
Closing discussion as completed.
